plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -971,12 +990,18 @@ protected void doPush(final String repository,
                     @Override
                     public void run() {
                         try {
+                            String digestPrefix = firstNonNull(tag, "latest") + ": digest: ";
                             ProgressStatus progressStatus;
                             while ((progressStatus = progressReader.next()) != null && exceptionHolder.get() == null) {
                                 progressMonitor.updateProgress(progressStatus);
                                 if (progressStatus.getError() != null) {
                                     exceptionHolder.set(progressStatus.getError());
                                 }
+                                String status = progressStatus.getStatus();
+                                if (status != null && status.startsWith(digestPrefix)) {
+                                        String digest = status.substring(digestPrefix.length(), status.indexOf(" ", digestPrefix.length()));
```

Comment:
To many spaces see [spec ](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#block-indentation-is-4-spaces)